OsmAnd/src/net/osmand/plus/views/MapControlsLayer.java

```diff
@@ -440,6 +444,10 @@ public void doNavigate() {
 		startNavigation();
 	}
 
+	public void showNavigationDialog() {
+		onNavigationClick();
```

Comment:
Dima said that the behavior when clicking on the navigation button and navigation in quick actions should be identical. Therefore I use the same method.